src/cells/vrnn_cell.py

```diff
@@ -3,7 +3,7 @@
 if sys.version_info[0] < 3:
   from models.tf_utils import *
 else:
-  from src.models.tf_utils import *
+  from models.tf_utils import *
```

Comment:
Otherwise it doesnt run at my computer. Double check if it still works at yours

Reply:
This usually has to do with which directory you run the python3 command in. Should not be a further issue I think